src/NServiceBus.Core/Reliability/Outbox/ForceBatchDispatchToBeIsolatedBehavior.cs

```diff
@@ -11,6 +11,11 @@ public Task Invoke(IBatchDispatchContext context, Func<IBatchDispatchContext, Ta
         {
             foreach (var operation in context.Operations)
             {
+                // Changing the dispatch consistency to be isolated to make sure the transport doesn't
+                // enlist the operations in the receive transaction. The transport might still want to batch
+                // operations for efficiency reasons but should never enlist in the incoming transport transaction.
```

Comment:
> The transport might still want to batch operations for efficiency reasons but should never enlist in the incoming transport transaction.

that does sound off given the public API is called "immediate dispatch". There shouldn't be any batching involved here since the user can assume the message to be physically sent when an "immediate dispatch" operation has completed?